models/percona_sso_model.go

```diff
@@ -29,22 +29,24 @@ import (
 // It is intended to have only one row in this table as PMM can be connected to Portal only once.
 //reform:percona_sso_details
 type PerconaSSODetails struct {
-	ClientID     string                 `reform:"client_id,pk"`
-	ClientSecret string                 `reform:"client_secret"`
-	IssuerURL    string                 `reform:"issuer_url"`
-	Scope        string                 `reform:"scope"`
-	AccessToken  *PerconaSSOAccessToken `reform:"access_token"`
+	ClientID             string                 `reform:"client_id,pk"`
+	ClientSecret         string                 `reform:"client_secret"`
+	IssuerURL            string                 `reform:"issuer_url"`
+	Scope                string                 `reform:"scope"`
+	AccessToken          *PerconaSSOAccessToken `reform:"access_token"`
+	PortalOrganizationID string                 `reform:"portal_organization_id"`
```

Comment:
oh you mean the fields other than `portal_organization_id`, I didn't notice that. @maxkondr should I change the names of all columns according to the tech spec or should we rename it in the spec without `portal_` prefix?

Reply:
I think it's more reasonable just to change docs. 
Actually all names are different, even `portal_organization_id` is spelled differenlty - in docs it has `portal_organization_id` with "s" in the middle (but I think spelling with "z" is better because we have it like this across the whole project)

Reply:
done